script/prove-block.sh

```diff
@@ -33,8 +33,10 @@ elif [ "$chain" == "taiko_a7" ]; then
 	l1Rpc="https://ethereum-holesky-rpc.publicnode.com"
 	beaconRpc="https://eth-holesky-beacon.public.blastapi.io"
 else
-	echo "Invalid chain name. Please use 'ethereum', 'holesky' or 'taiko_a7'."
-	exit 1
+	echo "Using customized chain name $1. Please double check the RPCs."
+	rpc="https://rpc.hekla.taiko.xyz"
+	l1Rpc="https://ethereum-holesky-rpc.publicnode.com"
+	beaconRpc="https://eth-holesky-beacon.public.blastapi.io"
```

Comment:
All rpc's can now be removed here I guess? 

Maybe now we can always add `-chain_spec_path chain_spec_list_default.json` so it reads in the latest json file without having to recompile.

Reply:
Yes, still refining. some of info previously are in wrong chain spec like `genesis time` due to legacy problem, I am re-org it, and also now we can pass down l1_chainspec directly rather than 2 l1_rpc strings.

Reply:
We can just default to that param inside of the `Cli` struct and then have people pass it only if customization is needed

Reply:
no need in CLI any more, now it's part of the chain spec, so by default every chain (should) has its own rpcs.

Reply:
I was referring to Brecht's `-chain_spec_path chain_spec_list_default.json` comment

Reply:
Oh, yes, I mistakingly thought you mean the rpc. 
For the CLI parameter, I chose option<_> because I believe people should be aware that they are overriding values, based on my own preference for being explicit :)